0009-Archivematica-original-design-principles.md

```diff
@@ -0,0 +1,49 @@
+# Archivematica original design principles
+
+* Status: accepted
+
+## Authors
+
+* Peter Van Garderen
+* Evelyn McLellan
+
+## Context and problem statement
+
+There is no open-source software system capable of implementing the functional requirements of the OAIS reference model (ISO 14721:2003). Digital preservation specialists must use multiple tools, which can be difficult to install and use, to perform discrete preservation actions. Those tools produce metadata that do not conform to digital preservation and exchange standards and schemas, and do not provide a way to automatically generate standardized, system-independent, self-documenting Archival Information Packages (AIPs) that package content and Preservation Description Information (PDI) as described by OAIS. Repository applications such as Fedora are capable of performing some but not all OAIS preservation actions, and tend to be complex to develop and maintain, posing a risk to future retrieval and readability of the AIPs. In fact, any middleware repository or database software that is required to access and read AIPs is inherently a risk to their long-term usability.
+
+## Decision drivers
+
+Artefactual designed an open-source, web-based archival description and access system called ICA-AtoM (Access To Memory) that has a broad user base around the world. ICA-AtoM does not provide digital preservation functionality as described by OAIS. It would benefit ICA-AtoM users to be able to integrate with a back-end system designed to preserve digital objects that are linked to access copies in ICA-AtoM. The system should also be usable on its own or in conjunction with other access tools.
```

Comment:
The broader use of Archivematica came up in a meeting earlier and it made me wonder if perhaps the problem statement could be expanded to describe its interoperability with a wide range (read: any) archival management system/content management system, etc.